lua/entities/gmod_wire_graphics_tablet.lua

```diff
@@ -25,7 +33,7 @@ if CLIENT then
 	function ENT:Draw()
 		self:DrawModel()
 
-		local draw_background = self:GetNetworkedBeamBool("draw_background", true)
+		local draw_background = self:GetDrawBackground()
```

Comment:
eh? where does that method come from?
if it's garry's, did it ever make sense to manually network it?